src/core/hle/service/hid/controllers/xpad.h

```diff
@@ -28,6 +29,67 @@ class Controller_XPad final : public ControllerBase {
     void OnLoadInputDevices() override;
 
 private:
+    struct Attributes {
+        union {
+            s32_le raw{};
+            BitField<0, 1, u32> IsConnected;
```

Comment:
```suggestion
            BitField<0, 1, u32> is_connected;
```

etc.

All struct members should be snake_case.